crates/router/src/core/payment_link.rs

```diff
@@ -396,17 +393,15 @@ pub async fn initiate_payment_link_flow(
     key_store: domain::MerchantKeyStore,
     merchant_id: common_utils::id_type::MerchantId,
     payment_id: common_utils::id_type::PaymentId,
-    request_headers: &header::HeaderMap,
+    _request_headers: &header::HeaderMap,
```

Comment:
We can remove the unused `request_headers` argument here and in `get_payment_link_status()` (both `v1` and `v2` features).